bots/pr/src/main/java/org/openjdk/skara/bots/pr/CSRCommand.java

```diff
@@ -54,9 +55,9 @@ private static void jbsReply(PullRequest pr, PrintWriter writer) {
                       "an issue in JBS, please use the `/issue` command in a comment in this pull request.");
     }
 
-    private static void linkReply(PullRequest pr, Issue issue, PrintWriter writer) {
-        writer.println("@" + pr.author().username() + " please create a [CSR](https://wiki.openjdk.org/display/csr/Main) request for issue " +
-                "[" + issue.id() + "](" + issue.webUrl() + ") with the correct fix version. " +
+    private static void csrCreateReply(PullRequest pr,  PrintWriter writer) {
+        writer.println("@" + pr.author().username() + " please create a [CSR](https://wiki.openjdk.org/display/csr/Main) " +
+                "request for any issue associated with this pr with the correct fix version " +
                 "This pull request cannot be integrated until the CSR request is approved.");
```

Comment:
This message doesn't really make sense. If there is more than one issue associated with the pr, then I don't think there is any point printing this at all. We can't know which issue that needs a CSR, so the person issuing the command has to be responsible for clarifying that. The main point of this message is to link to the issue.

So, if there is only one issue (which will be the case in 99% of cases involving CSRs), print the old message, otherwise don't call this method, or have it do nothing.